__tests__/database.test.js

```diff
@@ -166,6 +166,62 @@ describe("GET comments", () => {
     })
 })
 
+
+describe("POST comments", () => {
+    const matchCommentObject = {
+        comment_id: expect.any(Number),
+        votes: 0,
+        created_at: expect.any(String),
+        author: 'rogersop',
+        body: 'Amazing article',
+        article_id: 2
+      }
+      const validComment = {
+        username: 'rogersop',
+        body: 'Amazing article'
+    }
+    test("POST /api/articles/2/comments with existing user and valid body returns a comment", () => {
+        return request(app)
+        .post('/api/articles/2/comments')
+        .send(validComment)
+        .expect(201)
+        .then(({body}) => {
+            expect(body[0]).toMatchObject(matchCommentObject)
+        })
+    })
+    test("After POST, comment is added to the comments table", () => {
```

Comment:
I see in order to do this test you've changed the beforeEach to a beforeAll, I would say there's no need to do this for this test and that getting a response back from the post request with a new comment with the expected id is sufficient, it will also potentially cause problems down the line later when we add more functionality like further posting and/or deleting/patching etc.